src/commands/mute.js

```diff
@@ -0,0 +1,138 @@
+const Discord = require('discord.js');
+const ms = require('ms');
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const mutes = require('../models/mutes');
+const db = require('../db');
+const util = require('../util');
+
+/**
+ *
+ * @param {Discord.CommandInteraction} interaction
+ */
+ async function muteHandler(interaction) {
+	await interaction.deferReply({
+		ephemeral: true
+	});
+
+	const guild = await interaction.guild.fetch();
+	const executingMember = await interaction.member.fetch();
+	const executor = executingMember.user;
+	const users = interaction.options.getString('users');
+	const timeamount = interaction.options.getString('time');
+	const mutedRoleId = db.getDB().get('roles.muted');
+    const mutedRole = mutedRoleId && await guild.roles.fetch(mutedRoleId);
+
+	const userIds = [...new Set(Array.from(users.matchAll(Discord.MessageMentions.USERS_PATTERN), match => match[1]))];
+
+	const muteListEmbed = new Discord.MessageEmbed();
+	muteListEmbed.setTitle('User Mute :zipper_mouth:');
+	muteListEmbed.setColor(0xFFA500);
+
+	for (const userId of userIds) {
+		const member = await interaction.guild.members.fetch(userId);
+        const user = member.user;
+
+        const eventLogEmbed = new Discord.MessageEmbed();
+
+		eventLogEmbed.setColor(0xF24E43);
+		eventLogEmbed.setDescription('――――――――――――――――――――――――――――――――――');
+		eventLogEmbed.setTimestamp(Date.now());
+		eventLogEmbed.setTitle('Event Type: _Member Muted_');
+		eventLogEmbed.setFields(
+			{
+				name: 'User',
+				value: `<@${user.id}>`
+			},
+			{
+				name: 'User ID',
+				value: user.id
+			},
+			{
+				name: 'Executor',
+				value: `<@${executor.id}>`
+			},
+			{
+				name: 'Executor User ID',
+				value: executor.id
+			},
+			{
+				name: 'Amount of time muted',
+				value: timeamount
+			}
+		);
+		eventLogEmbed.setFooter({
+			text: 'Pretendo Network',
+			iconURL: guild.iconURL()
+		});
+
+		await util.sendEventLogMessage(guild, null, eventLogEmbed);
+
+		const sendMemberEmbeds = [];
+
+		const muteEmbed = new Discord.MessageEmbed();
+
+		muteEmbed.setTitle('Punishment Details');
+		muteEmbed.setDescription('You have been muted in the Pretendo Network server. You may not speak inside the server at this time, and an appeal may not be possible\nYou may review the details of your mute below');
+		muteEmbed.setColor(0xF24E43);
+		muteEmbed.setTimestamp(Date.now());
+		muteEmbed.setAuthor({
+			name: `Muted by: ${executingMember.user.tag}`,
+			iconURL: executingMember.user.avatarURL()
+		});
+		muteEmbed.setFooter({
+			text: 'Pretendo Network',
+			iconURL: guild.iconURL()
+		});
+		muteEmbed.setFields({
+			name: 'Mute Time',
+			value: timeamount
+		});
+
+		sendMemberEmbeds.push(muteEmbed);
+
+		await member.send({
+			embeds: sendMemberEmbeds
+		}).catch(() => console.log('Failed to DM user'));
```

Comment:
The formatting for these lines is off, it's indented too far and doesn't line up